int16     Device_ID
int16     Command
float32   Motor_rpm
float32   Motor_pos
float32   Motor_torque
float32   Motor_voltage
float32   Motor_current